### PR TITLE
Consume api endpoints for ignored word persistence

### PR DESCRIPTION
**Pull request by WillCMcC**

### CODE REVIEW

File: bot-utils.js [hunk 1]
```diff
@@ -1,58 +1,54 @@
...
-    writeIgnoredWords(listOfIgnoredWords);
-  });
+let apimeHeaders = {
+  'Authorization':
+    "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJQSG1QTVBHU3JwNjdITllXVzVwSDd5emd5QmRqTG1NWHZjckFaR0RiZnRGeTR0Zml6Rm9LY1pqdTdFQmtEbzdLSFpLUEZKaGJaUG1GZiI6InQtYjhmY2EyOGYtYTdjYi00ZWY2LWI4NmEtYzE4YTc5NzMxZjUxIiwiSG12TE1nMnFlUDNuSzVWbTV0TVE3RFQzeVd2NmtIRXB1dkJZMmhnYWVSR0dMandTTjRzTDU4MmZXdU5ZM3p6UlY4aGZGSG96Q1BjbkQiOiI3NmExYzk0OS1hNDljLTQwYjgtYWVjOS0xNmVlMzA4OTMyZmMiLCJQVUhpOXdvcEh3VmZ4OUJQR1c1RGI1SEtob3lHUGJDb2pBaEFOSHlmb2lGNFRnRTJEQlNGbnR6RWh2bTdacGlTcVc2ekRaaUpvaHU0QiI6dHJ1ZSwiV0ZEWWROb3ZZb1F4dnJWeFI0dmFiQXVoc0g5dHRnbUszVVVod0hVOHd0RGR5alJnSlE0NFBCMjVSY0hZQUFERGtvZWdTZnU4ZXlCcVQiOnRydWUsIlk1NThkalNWeHN0ZWczUGtBaDhCUUVaeFBLVGJuc1FKaFcyOXVldER6U0JTM3o4NWtZeG5rRVZHRHBabW9EMlJLdXZ0NHI1ZENhbW93Ijp0cnVlLCJzaWduZWQiOnRydWUsImlhdCI6MTYwMTAwMTQ2NX0.yrD48Ogj-6ydFDu7ML-H7Jo8DNk_Ae3McHrGiBBMttM",
+}
+const ignore = async (words) => {
+  words = words.map(word => {return {word: word}});
+  try {
+    words = await await axios.post("https://apime.dev/api/cupbot_ignored_words", words , {
```
Comment (gortron) [inline bot-utils.js:19]: What in the sam-hell is _await await_?

File: bot-utils.js [hunk 2]
```diff
@@ -1,58 +1,54 @@
...
+      headers: apimeHeaders,
+    });
+  } catch (error) {
+    throw error;
```
Comment (gortron) [inline bot-utils.js:23]: Do these throws kill the bot process? If not, fine!

**Review by gortron — APPROVED** [review]

**Comment by gortron** [comment]
Looks like build is failing from `yarn`. Maybe remove the `package-lock.json`. If it still fails, try `yarn upgrade`.

File: bot-utils.js (hunk 1 above)
Comment (WillCMcC) [reply bot-utils.js:19]: oopsie

File: bot-utils.js (hunk 2 above)
Comment (WillCMcC) [reply bot-utils.js:23]: good catch 👍

**Pull request merged**